Configs/.config/waybar/config.jsonc

```diff
@@ -225,19 +246,102 @@
         "format-source": "",
         "format-source-muted": "",
         "on-click": "pavucontrol -t 4",
-        "on-click-middle": "~/.config/hyprdots/scripts/volumecontrol.sh -i m",
-        "on-scroll-up": "~/.config/hyprdots/scripts/volumecontrol.sh -i i",
-        "on-scroll-down": "~/.config/hyprdots/scripts/volumecontrol.sh -i d",
+        "on-click-middle": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -i m",
+        "on-scroll-up": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -i i",
+        "on-scroll-down": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -i d",
```

Comment:
same mistake  here

Reply:
> same mistake here

please check [this](https://github.com/ALEX5402/hyprdots/commit/c43a7e040a9d25dff0f60538527c1dca2d5c1a32
) commit that mistake was fixed now